sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/AsyncDocumentClient.java

```diff
@@ -660,7 +660,8 @@ public AzureKeyCredential getCredential() {
      * @param options        the query request options.
```

Comment:
nit: add classType in the param. Same comments for other places as well.

Reply:
Fixed in next iteration